
# ethernet raw tx
string   data_type
uint32   data_size
uint8[]  data